src/api/common/schemas/get-ledger.json

```diff
@@ -14,7 +14,12 @@
     "parentLedgerHash": {"$ref": "hash256"},
     "parentCloseTime": {"type": "integer", "minimum": 0},
     "totalDrops": {"$ref": "value"},
-    "transactionHash": {"$ref": "hash256"}
+    "transactionHash": {"$ref": "hash256"},
+    "transactions": {"type": "array", "items": {"type": "object"}},
+    "rawTransactions": {"type": "string"},
+    "transactionHashes": {"type": "array", "items": {"$ref": "hash256"}},
+    "rawAccounts": {"type": "string"},
```

Comment:
The `raw` forms will be used in `computeLedgerHash`. The convention is to keep rippled format objects as strings to emphasize that the user should never look at them. The `rawTransactions` string is mostly redundant with the `transactions` array. There is not yet a parsed version of `rawAccounts`, but that may be added later if demand exists.